samples/build_utils/CMakeLists.txt

```diff
@@ -81,6 +90,7 @@ macro(FindOpenCLHeaders TARGET)
     set(CMAKE_EXTRA_INCLUDE_FILES)
   endif()
   if (NOT (OPENCL_INC_FOUND AND CL_KERNEL_SUB_GROUP_INFO_SIZE))
+    RequirePythonInterp()
```

Comment:
Please move this line down, after set(), just for consistency

Reply:
:ok_hand: